File: APVN.LeadsPlatform.Models/LeadsFilterModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace APVN.LeadsPlatform.Models
{
    public class LeadsFilterModel : Pager
    {
        public string KeyWord { get; set; }
        public String DealerName { get; set; }
        public String DealerIds { get; set; }   // Danh sach id dearler 12,4,5
        public DateTime?[] CreatedDate { set; get; }
        public DateTime?[] StartCareDate { get; set; }
        public string[] CreatedDateArr { set; get; }
        public string[] StartCareDateArr { get; set; }
        public int? Status { get; set; }
        public int? MakeId { get; set; }
        public int? ModelId { get; set; }
        public int? SecondHand { get; set; }
        public int? CityId { get; set; }
        public int? AssignToId { get; set; }
        public int? IsActive { get; set; }
        public int? CampaignId { get; set; }
        public int? SourceLead { get; set; }
        public int? Source { get; set; }
        public int? BankId { get; set; }
        public int? PaymentStatus { get; set; }
        public int? Type { get; set; }

    }
}

File: APVN.LeadsPlatform.Client/apvn-leadsplatform-client/src/app/models/leads-filter-model.ts
import { LeadsManagermentResult } from './leadsmanagerment-search-result';
import { DatePipe } from '@angular/common';

export class LeadsFilterModel {
    KeyWord: string;
    DealerName: string;
    CreatedDate: Array<Date>;
    CreatedDateArr: Array<string>;
    StartCareDate: Array<Date>;
    StartCareDateArr: Array<string>;
    MakeId: number;
    ModelId: number;
    SecondHand: number;
    CityId: number;
    AssignToId: number;
    IsActive: number;
    CampaignId: number;
    SourceLead: number;
    Status: number;
    Type: number;
    BankId: number;
    PaymentStatus: number;

    pageIndex: number;
    CreateFromDate: DatePipe;
    pageSize: number = 10;
    // result: LeadsManagermentResult[];
    totalRecord: number;
}